cxx/hirm.hh

```diff
@@ -65,6 +65,11 @@ class HIRM {
 
   double logp_score() const;
 
```

Comment:
Please add brief descriptions of these methods.

Reply:
Done.